services/openshift/scripts/openshift_provision

```diff
@@ -21,9 +21,9 @@ wait_for_openshift_api() {
       ((ATTEMPT++))
     done
 
-    # Wait for API to be accessible
+    # Use health URL to check OpenShift readiness
     ATTEMPT=0
-    until curl -ksSf https://127.0.0.1:8443/api > /dev/null 2>&1 || [ $ATTEMPT -eq 30 ]; do
+    until curl -ksSf https://127.0.0.1:8443/healthz/ready > /dev/null 2>&1 || [ $ATTEMPT -eq 30 ]; do
```

Comment:
+1 

FYI - https://docs.openshift.com/enterprise/3.1/dev_guide/application_health.html
I was about to suggest to check for the return string 'ok', but I think a return code of 200 is fine (using `sSf` as curl options should take care of the command failing unless the error code is 200)